tests/vcr_python_wrapper.py

```diff
@@ -108,45 +108,44 @@ def filter_response(response):
         response['headers'].pop(header.lower(), None)
         response['headers'].pop(header, None)
     try:
-        json_sbody = json.loads(response['body']['string'])
-        if "host" in json_sbody:
-            json_sbody['host'] = 'FILTERED'
-        if "account" in json_sbody:
-            json_sbody['account'] = 'FILTERED'
-        if "base_dn" in json_sbody:
-            json_sbody['base_dn'] = 'FILTERED'
-        if "groups_base" in json_sbody:
-            json_sbody['groups_base'] = 'FILTERED'
-        response['body']['string'] = json.dumps(json_sbody)
-    except Exception:
-        pass
-    # another for the search result
-    try:
-        json_sbody = json.loads(response['body']['string'])
-        if "host" in json_sbody['results'][0]:
-            json_sbody['results'][0]['host'] = 'FILTERED'
-        if "account" in json_sbody['results'][0]:
-            json_sbody['results'][0]['account'] = 'FILTERED'
-        if "base_dn" in json_sbody['results'][0]:
-            json_sbody['results'][0]['base_dn'] = 'FILTERED'
-        if "groups_base" in json_sbody['results'][0]:
-            json_sbody['results'][0]['groups_base'] = 'FILTERED'
-            response['body']['string'] = json.dumps(json_sbody)
-    except Exception:
+        json_body = json.loads(response['body']['string'])
+    except json.JSONDecodeError:
         pass
+    else:
+        saved_str = response['body']['string']
```

Comment:
This is used nowhere, right?

Reply:
I think so

Reply:
Removed it

Reply:
Unfortunately, I cannot identify the errors that the linter outputs.